serverless.yaml

```diff
@@ -12,6 +12,11 @@ custom:
   exportRequestTableJobStatusIndex: 'jobStatus-index'
   stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
+  lambdaLatencyThreshold: ${env:lambdaLatencyThreshold}
+  apigatewayServerErrorThreshold: ${env:apigatewayServerErrorThreshold}
+  apigatewayBadRequestErrorThreshold: ${env:apigatewayBadRequestErrorThreshold}
+  lambdaErrorThreshold: ${env:lambdaErrorThreshold}
+  DDBToESLambdaErrorThreshold: ${env:DDBToESLambdaErrorThreshold}
```

Comment:
opt doesn't work ? That's so weird.

Reply:
Me too. I can't figure out why this would be the case. You shouldn't be hitting some parameter option limit here.

Reply:
I was suggested starting with v3 release, such CLI options won't be supported at all. It is recommended to use env vars instead by the serverless support team.

Reply:
serverless v3 doesn't allow opt any more. We are using serverless v2 which supports opts. You decided go with env for future compatibility.

Reply:
Ah, thanks for the clarification.